Update analytics docs (#794)
hasura/ddn-docs: docs/observability/model-analytics/field.mdx

By tracking the frequency and context of field queries, you can make informed decisions about API evolution, performance
optimization, and data governance.

> Field analytics can be accessed via the `Analytics` tab within the `Explorer` panel for any specific model or command, allowing users to view data for either an entire project or a single build

Reviewer: ```suggestion
Field analytics can be accessed via the `Analytics` tab within the `Explorer` panel for any specific model or command, allowing users to view data for either the project API or a of a single build
```
